src/runtime-rs/crates/shim-ctl/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "shim-ctl"
```

Comment:
I like the idea of adding a new tool for this, but maybe a different PR entirely?